### PR TITLE
Fix inventory information display

### PR DESCRIPTION
**Pull request by trasher**
Display message when no agent has been linked, not all asset have a last_inventory_update

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

File: templates/components/form/inventory_info.html.twig [hunk 1]
```diff
@@ -152,11 +152,17 @@
          });
       });
       </script>
+   {%  else %}
+   <div class="card-body row">
+      <div class="mb-3 col-12 col-sm-6">
+         {{ __('No agent has been linked.') }}
```
Comment (cedric-anne) [inline templates/components/form/inventory_info.html.twig:158]: We may have a linked agent in the past that has been removed due to stale agents configuration, or cleaning from some plugin. Maybe a vague sentence would be more appropriate.
```suggestion
         {{ __('Agent information is not available.') }}
```

**Review by cedric-anne — APPROVED** [review]

File: templates/components/form/inventory_info.html.twig [hunk 2]
```diff
@@ -152,11 +152,17 @@
          });
       });
       </script>
+   {%  else %}
```
Comment (cedric-anne) [inline templates/components/form/inventory_info.html.twig:155]:
```suggestion
   {% else %}
```

File: templates/components/form/inventory_info.html.twig (hunk 1 above)
Comment (trasher) [reply templates/components/form/inventory_info.html.twig:158]: I'm not in favor of introducing a new string few days before release. Also, if we change here, it must also be changed in PHP feature file.

Comment (g-bougard) [reply templates/components/form/inventory_info.html.twig:158]: Another suggestion if a shorter sentence is better.
```suggestion
         {{ __('No agent linked') }}
```

**Pull request merged**